src/pages/Dashboard/Markets/BorrowMarket/BorrowingTable.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useMemo } from 'react';
+import { Table, ITableProps, Token } from 'components';
+import { useTranslation } from 'translation';
+import { Asset, TokenId } from 'types';
+import {
+  formatCoinsToReadableValue,
+  formatCentsToReadableValue,
+  formatApy,
+} from 'utilities/common';
+
+export interface IBorrowingUiProps {
+  assets: Asset[];
+  isXvsEnabled: boolean;
+}
+
+const BorrowingTable: React.FC<IBorrowingUiProps> = ({ assets, isXvsEnabled }) => {
+  const { t } = useTranslation();
+  const columns = useMemo(
+    () => [
+      { key: 'asset', label: t('markets.columns.asset'), orderable: false },
+      { key: 'apyEarned', label: t('markets.columns.apyEarned'), orderable: true },
+      { key: 'balance', label: t('markets.columns.balance'), orderable: true },
+      { key: 'percentOfLimit', label: t('markets.columns.percentOfLimit'), orderable: true },
+    ],
+    [],
+  );
+  // Format assets to rows
+  const rows: ITableProps['data'] = assets.map(asset => {
+    const borrowApy = isXvsEnabled ? asset.xvsBorrowApy.plus(asset.borrowApy) : asset.borrowApy;
+
+    return [
+      {
+        key: 'asset',
+        render: () => <Token symbol={asset.symbol as TokenId} />,
+        value: asset.id,
+      },
+      {
+        key: 'apyEarned',
+        render: () => formatApy(borrowApy),
+        value: borrowApy.toNumber(),
+      },
+      {
+        key: 'balance',
+        render: () =>
+          formatCoinsToReadableValue({
+            value: asset.borrowBalance,
+            tokenId: asset.id as TokenId,
+          }),
+        value: asset.borrowBalance.toString(),
+      },
+      {
+        key: 'percentOfLimit',
+        // Convert liquidity (expressed in dollars) to cents, then format it to
+        // readable value
+        render: () =>
+          formatCentsToReadableValue({
+            value: asset.liquidity.multipliedBy(100),
+          }),
```

Comment:
This should display a progress bar next to a value (we now have a `PercentageRepresentation` component for that):
<img width="188" alt="Screen Shot 2022-04-26 at 22 08 40" src="https://user-images.githubusercontent.com/44675210/165392720-0de914a5-fd4c-4c64-a3b6-5822aed8f7e8.png">